### DISCUSSION TITLE
How to make a dashboard in Python like Shiny

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#651] [Tips]
Saya memiliki dashboard seperti ini di R Shiny : 

<img width="917" alt="image" src="https://github.com/teamalgoritma/community/assets/106010284/f6daf617-dfda-40a0-b338-c20cdd1c1870">

Saya ingin membuat dashboardnya di python. Apakah ada referensi yang bisa diberikan ?

Terima kasih sebelumnya kak 🙏

**Comment by idoali** [comment]
Ada banyak pilihan pembuatan dashboard di Python pak. Salah satunya adalah streamlit yang bisa dikatakan cukup mudah dipelajari. Berikut linknya jiika bapak ingin mencoba https://streamlit.io/

Semoga dapat membantu pak

**Reply by yohanespm77** [reply]
Kalau pakai streamlit apakah tampilannya bisa seperti navbarpage di shiny kak ?

**Reply by jafarhf1** [reply]
Dear Pak @yohanespm77,

Apabila tampilan yang Bapak maksud adalah navbar yang berada di atas dashboardnya, maka itu juga bisa dilakukan menggunakan streamlit contohnya seperti pada link berikut: https://github.com/BugzTheBunny/streamlit_navbar.

Semoga membantu pak, Terima kasih

**Reply by yohanespm77** [reply]
Saya coba run, tapi error kak @jafarhf1 

<img width="782" alt="image" src="https://github.com/teamalgoritma/community/assets/106010284/10926c05-9a02-465c-8419-73990bd0664a">


Mohon bantuannya kak 🙏

**Reply by jafarhf1** [reply]
Dear Pak @yohanespm77,

Berikut terdapat link diskusi mengenai error tersebut: https://github.com/BugzTheBunny/streamlit_navbar/issues/1

Semoga membantu ya pak, Terima kasih

**Comment by kinantytasyao** [comment]
Dear @yohanespm77 ,
Untuk menjaga agar forum diskusi berjalan kondusif, kami izin melakukan close pada diskusi ini karena belum ada respon setelah lebih dari 7 hari respon terakhir. Semoga dapat dipahami. Terima kasih.
